### DISCUSSION TITLE
No menu open On search typing

### DISCUSSION THREAD

**Discussion by cubecleveland** [JedWatson/react-select#4885] [Q&A]
Hello,

I have the following scnarion i want the select box to serve as a search bar with 1 label.
My thought is to have the Selectt serachable and  selectable (1 options only)
But when someone searches i want the menu to stay closed as this indicates a regular searching that will do abackground API fecth?  Am i going in the right direction or are there any ways to implement this behaviour ?

**Comment by ebonow** [comment]
There is a `menuIsOpen` prop that you can use to enable and disable the menu from being shown. Please note that setting this prop to `undefined` instead of `false` will allow it to be opened again later when the user clicks on the down arrow, whereas false will override the default open menu behavior.

**Reply by cubecleveland** [reply]
Thanks.
Im struggling applying th behaviour that we need.
Is that something tht can be achived via the component or will not be a good use case?

Basically we are trying to make a filter that will drop down a suggestion box and then will use the options as tags then handle the other input portion as the 2nd argument to an api call...
Thanks!
